framework/utils/random-string.ts

```diff
@@ -2,12 +2,21 @@
 
 const randomCharacters = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
 
-export function randomString(length: number, base = randomCharacters.length): string {
-  if (base > randomCharacters.length || base <= 0) base = randomCharacters.length;
+const randomCharactersLowercase = 'abcdefghijklmnopqrstuvwxyz0123456789';
```

Comment:
This was needed because collection names can only have numbers underscores and lowercase characters.